golang/cosmos/x/vpurse/module.go

```diff
@@ -144,10 +145,11 @@ func (am AppModule) EndBlock(ctx sdk.Context, req abci.RequestEndBlock) []abci.V
 			for _, attr := range event.Attributes {
 				// fmt.Println("have transfer attr", string(attr.GetKey()), string(attr.GetValue()))
 				switch string(attr.GetKey()) {
-				case "recipient":
-				case "sender":
+				case "recipient", "sender":
 					address := string(attr.GetValue())
-					ensureBalanceIsPresent(address)
+					if err := ensureBalanceIsPresent(address); err != nil {
+						fmt.Println("Cannot ensure vpurse balance for", address, err)
```

Comment:
Is there a more appropriate action for the error, e.g. some sort of logging other than stdout?

Reply:
I used the "log" package.  It may still write to `stdout`, but that's what the entire Cosmos-SDK does too.